elasticsearch/jvmopts.sls

```diff
@@ -11,7 +11,7 @@ include:
     - mode: 0770
     - user: elasticsearch
     - group: elasticsearch
-    - contents: {{ jvm_opts }}
+    - contents: {{ jvm_opts | json }}
```

Comment:
@myii is it ok to use `json` or do we still need to do `tojson` for filtering?

Reply:
`json` is needed for the time being, until we no longer need to support `2017.7` -- i.e. until the next major release.